did-eth-method-draft.md

```diff
@@ -16,15 +16,51 @@
 ## Design Goals
 
 
-## Identifier scheme
+## DID Method Name
 
-### Syntax and Interpretation
+The namestring that shall identify this DID method is: `eth`
 
-```
-eth-did    = "did:eth:<chain_id>:" address
-chain_id   = chain_id according to [CAIP-2]
-address    = account_id according to [CAIP-10]
-```
+A DID that uses this method MUST begin with the following prefix: `did:eth`. Per the DID specification, this string
+MUST be in lowercase. The remainder of the DID, after the prefix, is specified below.
+
+## Method Specific Identifier
+
+The method specific identifier is represented as the HEX-encoded secp256k1 public key (in compressed form),
+or the corresponding HEX-encoded Ethereum address on the target network, prefixed with `0x`.
+
+    eth-did = "did:ethr:" eth-specific-identifier
+    eth-specific-identifier = [ ethr-network ":" ] ethereum-address / public-key-hex / ens-name
+    eth-network = "mainnet" / "goerli" / network-chain-id
+    network-chain-id = "0x" *HEXDIG
```

Comment:
do we want to keep the hex encoding for chain IDs? (can be raised as an issue and discussed separately